diff --git a/0001-add-font-configuration.path b/0001-add-font-configuration.path
new file mode 100644
index 0000000..b0e8cbd
--- /dev/null
+++ b/0001-add-font-configuration.path
@@ -0,0 +1,133 @@
+diff --git a/chrome/userChrome.css b/chrome/userChrome.css
+index 1cc831a..b36398c 100644
+--- a/chrome/userChrome.css
++++ b/chrome/userChrome.css
+@@ -1,9 +1,9 @@
+-/*
++/*
+ ┌─┐┬┌┬┐┌─┐┬  ┌─┐
+-└─┐││││├─┘│  ├┤
++└─┐││││├─┘│  ├┤
+ └─┘┴┴ ┴┴  ┴─┘└─┘
+-┌─┐┌─┐─┐ ┬
+-├┤ │ │┌┴┬┘
++┌─┐┌─┐─┐ ┬
++├┤ │ │┌┴┬┘
+ └  └─┘┴ └─
+
+ by Miguel Avila
+@@ -11,7 +11,7 @@ by Miguel Avila
+ */
+
+ /*
+-
++
+ ┌─┐┌─┐┌┐┌┌─┐┬┌─┐┬ ┬┬─┐┌─┐┌┬┐┬┌─┐┌┐┌
+ │  │ ││││├┤ ││ ┬│ │├┬┘├─┤ │ ││ ││││
+ └─┘└─┘┘└┘└  ┴└─┘└─┘┴└─┴ ┴ ┴ ┴└─┘┘└┘
+@@ -23,6 +23,20 @@ by Miguel Avila
+   --sfsecondary: #201e21;
+ }
+
++:root {
++  --sfwindow: #${background-darker};
++  --sfsecondary: #${background};
++  --mff-tab-font-family: "Lato";
++  --mff-tab-font-size: 11pt;
++  --mff-tab-font-weight: 400;
++  --mff-urlbar-font-family: "Lato";
++  --mff-urlbar-font-size: 11pt;
++  --mff-urlbar-font-weight: 700;
++  --mff-urlbar-results-font-family: "Lato";
++  --mff-urlbar-results-font-size: 11pt;
++  --mff-urlbar-results-font-weight: 700;
++}
++
+ /* Urlbar View */
+
+ /*─────────────────────────────*/
+@@ -36,10 +50,10 @@ by Miguel Avila
+
+ /*─────────────────────────────*/
+
+-/*
++/*
+ ┌─┐┌─┐┬  ┌─┐┬─┐┌─┐
+ │  │ ││  │ │├┬┘└─┐
+-└─┘└─┘┴─┘└─┘┴└─└─┘
++└─┘└─┘┴─┘└─┘┴└─└─┘
+ */
+
+ /* Tabs colors  */
+@@ -47,12 +61,12 @@ by Miguel Avila
+   > #tabbrowser-arrowscrollbox
+   > .tabbrowser-tab
+   > .tab-stack
+-  > .tab-background[multiselected='true'],
++  > .tab-background[multiselected="true"],
+ #tabbrowser-tabs:not([movingtab])
+   > #tabbrowser-arrowscrollbox
+   > .tabbrowser-tab
+   > .tab-stack
+-  > .tab-background[selected='true'] {
++  > .tab-background[selected="true"] {
+   background-image: none !important;
+   background-color: var(--toolbar-bgcolor) !important;
+ }
+@@ -76,11 +90,11 @@ by Miguel Avila
+   background-color: var(--sfwindow) !important;
+ }
+
+-/*
++/*
+
+-┌┬┐┌─┐┬  ┌─┐┌┬┐┌─┐
+- ││├┤ │  ├┤  │ ├┤
+-─┴┘└─┘┴─┘└─┘ ┴ └─┘
++┌┬┐┌─┐┬  ┌─┐┌┬┐┌─┐
++ ││├┤ │  ├┤  │ ├┤
++─┴┘└─┘┴─┘└─┘ ┴ └─┘
+ ┌─┐┌─┐┌┬┐┌─┐┌─┐┌┐┌┌─┐┌┐┌┌┬┐┌─┐
+ │  │ ││││├─┘│ ││││├┤ │││ │ └─┐
+ └─┘└─┘┴ ┴┴  └─┘┘└┘└─┘┘└┘ ┴ └─┘
+@@ -96,7 +110,7 @@ by Miguel Avila
+   display: none !important;
+ }
+
+-#nav-bar:not([tabs-hidden='true']) {
++#nav-bar:not([tabs-hidden="true"]) {
+   box-shadow: none;
+ }
+
+@@ -117,13 +131,13 @@ by Miguel Avila
+   margin-left: -4px !important;
+ }
+
+-.tabbrowser-tab:is([visuallyselected='true'], [multiselected])
++.tabbrowser-tab:is([visuallyselected="true"], [multiselected])
+   > .tab-stack
+   > .tab-background {
+   box-shadow: none !important;
+ }
+
+-.tabbrowser-tab[last-visible-tab='true'] {
++.tabbrowser-tab[last-visible-tab="true"] {
+   padding-inline-end: 0 !important;
+ }
+
+@@ -141,7 +155,7 @@ by Miguel Avila
+   margin-left: 0 !important;
+ }
+
+-#urlbar[focused='true'] > #urlbar-background {
++#urlbar[focused="true"] > #urlbar-background {
+   box-shadow: none !important;
+ }
+
+@@ -202,4 +216,4 @@ toolbarbutton.bookmark-item:not(.subviewbutton) {
+ #context_selectAllTabs,
+ #context_closeTabOptions {
+   display: none !important;
+-}
+\ No newline at end of file
++}
diff --git a/chrome/userChrome.css b/chrome/userChrome.css
index 1cc831a..b36398c 100644
--- a/chrome/userChrome.css
+++ b/chrome/userChrome.css
@@ -1,9 +1,9 @@
-/*
+/*
 ┌─┐┬┌┬┐┌─┐┬  ┌─┐
-└─┐││││├─┘│  ├┤
+└─┐││││├─┘│  ├┤
 └─┘┴┴ ┴┴  ┴─┘└─┘
-┌─┐┌─┐─┐ ┬
-├┤ │ │┌┴┬┘
+┌─┐┌─┐─┐ ┬
+├┤ │ │┌┴┬┘
 └  └─┘┴ └─

 by Miguel Avila
@@ -11,7 +11,7 @@ by Miguel Avila
 */

 /*
-
+
 ┌─┐┌─┐┌┐┌┌─┐┬┌─┐┬ ┬┬─┐┌─┐┌┬┐┬┌─┐┌┐┌
 │  │ ││││├┤ ││ ┬│ │├┬┘├─┤ │ ││ ││││
 └─┘└─┘┘└┘└  ┴└─┘└─┘┴└─┴ ┴ ┴ ┴└─┘┘└┘
@@ -23,6 +23,20 @@ by Miguel Avila
   --sfsecondary: #201e21;
 }

+:root {
+  --sfwindow: #${background-darker};
+  --sfsecondary: #${background};
+  --mff-tab-font-family: "Lato";
+  --mff-tab-font-size: 11pt;
+  --mff-tab-font-weight: 400;
+  --mff-urlbar-font-family: "Lato";
+  --mff-urlbar-font-size: 11pt;
+  --mff-urlbar-font-weight: 700;
+  --mff-urlbar-results-font-family: "Lato";
+  --mff-urlbar-results-font-size: 11pt;
+  --mff-urlbar-results-font-weight: 700;
+}
+
 /* Urlbar View */

 /*─────────────────────────────*/
@@ -36,10 +50,10 @@ by Miguel Avila

 /*─────────────────────────────*/

-/*
+/*
 ┌─┐┌─┐┬  ┌─┐┬─┐┌─┐
 │  │ ││  │ │├┬┘└─┐
-└─┘└─┘┴─┘└─┘┴└─└─┘
+└─┘└─┘┴─┘└─┘┴└─└─┘
 */

 /* Tabs colors  */
@@ -47,12 +61,12 @@ by Miguel Avila
   > #tabbrowser-arrowscrollbox
   > .tabbrowser-tab
   > .tab-stack
-  > .tab-background[multiselected='true'],
+  > .tab-background[multiselected="true"],
 #tabbrowser-tabs:not([movingtab])
   > #tabbrowser-arrowscrollbox
   > .tabbrowser-tab
   > .tab-stack
-  > .tab-background[selected='true'] {
+  > .tab-background[selected="true"] {
   background-image: none !important;
   background-color: var(--toolbar-bgcolor) !important;
 }
@@ -76,11 +90,11 @@ by Miguel Avila
   background-color: var(--sfwindow) !important;
 }

-/*
+/*

-┌┬┐┌─┐┬  ┌─┐┌┬┐┌─┐
- ││├┤ │  ├┤  │ ├┤
-─┴┘└─┘┴─┘└─┘ ┴ └─┘
+┌┬┐┌─┐┬  ┌─┐┌┬┐┌─┐
+ ││├┤ │  ├┤  │ ├┤
+─┴┘└─┘┴─┘└─┘ ┴ └─┘
 ┌─┐┌─┐┌┬┐┌─┐┌─┐┌┐┌┌─┐┌┐┌┌┬┐┌─┐
 │  │ ││││├─┘│ ││││├┤ │││ │ └─┐
 └─┘└─┘┴ ┴┴  └─┘┘└┘└─┘┘└┘ ┴ └─┘
@@ -96,7 +110,7 @@ by Miguel Avila
   display: none !important;
 }

-#nav-bar:not([tabs-hidden='true']) {
+#nav-bar:not([tabs-hidden="true"]) {
   box-shadow: none;
 }

@@ -117,13 +131,13 @@ by Miguel Avila
   margin-left: -4px !important;
 }

-.tabbrowser-tab:is([visuallyselected='true'], [multiselected])
+.tabbrowser-tab:is([visuallyselected="true"], [multiselected])
   > .tab-stack
   > .tab-background {
   box-shadow: none !important;
 }

-.tabbrowser-tab[last-visible-tab='true'] {
+.tabbrowser-tab[last-visible-tab="true"] {
   padding-inline-end: 0 !important;
 }

@@ -141,7 +155,7 @@ by Miguel Avila
   margin-left: 0 !important;
 }

-#urlbar[focused='true'] > #urlbar-background {
+#urlbar[focused="true"] > #urlbar-background {
   box-shadow: none !important;
 }

@@ -202,4 +216,4 @@ toolbarbutton.bookmark-item:not(.subviewbutton) {
 #context_selectAllTabs,
 #context_closeTabOptions {
   display: none !important;
-}
\ No newline at end of file
+}
